package.json

```
@@ -32,70 +32,72 @@
+    "@react-native-picker/picker": "2.11.1",
     "@react-navigation/native": "^7.0.14",
     "@superwall/react-native-superwall": "^2.0.0",
     "convex": "^1.19.2",
     "date-fns": "^4.1.0",
-    "expo": "~52.0.47",
-    "expo-camera": "~16.0.18",
-    "expo-clipboard": "~7.0.1",
-    "expo-constants": "~17.0.8",
-    "expo-crypto": "~14.0.2",
+    "expo": "~53.0.0",
+    "expo-auth-session": "~6.3.15",
```

Biome (2.1.2)

[error] The key expo-auth-session was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)